Documentation/git.txt

```diff
@@ -342,7 +342,7 @@ Repository, command and file interfaces
 ---------------------------------------
```

Comment:
Sorry, I didn’t notice this document before. I will take a careful look at it later.

Reply:
> Sorry, I didn’t notice this document before. I will take a careful look at it later.

Please reply on the list, otherwise Junio won't see your reply. (Follow the "reply to this" link [in this comment](https://github.com/git/git/pull/1669#pullrequestreview-1994920107) to find out more.)

Reply:
> > Sorry, I didn’t notice this document before. I will take a careful look at it later.
> 
> Please reply on the list, otherwise Junio won't see your reply. (Follow the "reply to this" link [in this comment](https://github.com/git/git/pull/1669#pullrequestreview-1994920107) to find out more.)

Thank you for the reminder. I'm having issues with my email account today and am unable to log in using a third-party client. I will try to reply to this message tomorrow.

Reply:
If you have a way to import mbox files into your email account, that's another option. For example, appending `/raw` to https://lore.kernel.org/git/xmqqpluvu5vk.fsf@gitster.g/ (like [so](https://lore.kernel.org/git/xmqqpluvu5vk.fsf@gitster.g/raw)) will give you that mbox file with all the information your mail program should require to reply.

Reply:
(Also, you should have those mails in your inbox anyway, no? Or did you provide an incorrect email address in the commits' author information?)

Reply:
Thank you for your reply. I have replied to that message via Thunderbird, and I hope Junio can receive it.

Reply:
I do not yet see it here: https://lore.kernel.org/git/xmqqpluvu5vk.fsf@gitster.g/. Sometimes mails get delayed, but it could also be the very, _very_ common issue that _the Git mailing list silently drops HTML mails_! So unless you have configured your Thunderbird to send plain-text-only mails, it might have fallen prey to this issue. Git has [a section on configuring Thunderbird](https://github.com/git/git/blob/v2.44.0/Documentation/git-format-patch.txt#L561-L569) that might, or might not, be totally outdated.

Reply:
> I do not yet see it here: https://lore.kernel.org/git/xmqqpluvu5vk.fsf@gitster.g/. Sometimes mails get delayed, but it could also be the very, _very_ common issue that _the Git mailing list silently drops HTML mails_! So unless you have configured your Thunderbird to send plain-text-only mails, it might have fallen prey to this issue. Git has [a section on configuring Thunderbird](https://github.com/git/git/blob/v2.44.0/Documentation/git-format-patch.txt#L561-L569) that might, or might not, be totally outdated.

It seems I've succeeded. Do I need to reply to Martin Ågren via email again?